src/graph/service/PermissionCheck.cpp

```diff
@@ -77,6 +77,7 @@ namespace graph {
     case Sentence::Kind::kCreateEdge:
     case Sentence::Kind::kAlterEdge:
     case Sentence::Kind::kDropTag:
+    case Sentence::Kind::kIsomor:
```

Comment:
The handling function for these cases are `canWriteSchema`.  You don't need to write schema. How about trying to add the case of Isomor around line 130 so `canReadSchemaOrData` is the handling function?